### ISSUE TITLE
Pac4jScalaTemplateHelper with PlayCookieSessionStore returns no profile

### ISSUE THREAD

**Issue by rebstar6** [pac4j/play-pac4j#257]
the Pac4jScalaTemplateHelper seems to explicitly specify the PlayCacheSessionStore. I setup my app using the PlayCookieSessionStore and noticed that on calls to the following:

```
val cacheProf = pac4jScalaTemplateHelper.getCurrentProfile(request)

val webContext = new PlayWebContext(request, playSessionStore)
val profileManager = new ProfileManager[CommonProfile](webContext)
val cookieProf = toScala(profileManager.get(true))
```

cacheProf is None, but cookieProf contains the current profile.

I got stuck on this for a while, so figured at least some more docs on this point could help others!

**Comment by leleuj** [comment]
I don't think we need more documentation here, it's a bug, we should retrieve the appropriate `PlaySessionStore`, whether it is a `PlayCacheSessionStore` or a `PlayCookieSessionStore`.

Feel free to submit a contribution. Thanks.

**Comment by leleuj** [comment]
Fixed in the incoming v7.0.1 release.
